public/app/plugins/datasource/grafana/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { DataFrame, DataFrameJSON, dataFrameToJSON } from '@grafana/data';
+import { GRAFANA_DATASOURCE_NAME } from 'app/features/alerting/unified/utils/datasource';
+import { PanelModel } from 'app/features/dashboard/state';
+
+import { GrafanaQuery, GrafanaQueryType } from './types';
+
+export const changeToSnapshotData = (frames: DataFrame[], panel: PanelModel) => {
```

Comment:
@torkelo I was thinking that maybe this would sit better in a place like `public/app/core/utils/changeToSnapshotData.ts` instead of the grafana datasource. Wdyt?